runtime/altair/src/lib.rs

```diff
@@ -72,10 +70,14 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("altair"),
 	impl_name: create_runtime_str!("altair"),
 	authoring_version: 1,
-	spec_version: 1008,
+	spec_version: 1900,
```

Comment:
Why the logical change here? What does it represent or is it a typo?

Reply:
This matches how parity does their versioning - polkadot X.Y turns into runtime XY00. I'm not sure we need to do this, and I'm personally happy to just keep incrementing a release number without trying to tie it to any more formal versioning structure.

Reply:
Since we are tied to updating the node as well and that follows a semver X.Y.Z. One reason I thought about using the same as parity is bc if we dont, then our x.y.z for 1008, would be 1.0.8, which puts us in version 1 already, which prob we are not ready for yet. Alternatively we can use 0.10.08 as the equivalent of 1008, I can change that too

Reply:
Like the later more, but I am happy with both.